File: test/export/pas/02-entrada-salida.pas
Program sin_titulo;
Var
	a: Integer;
	aux_leer_logica: String;
	b: Real;
	c: String;
	d: Boolean;
Begin
	ReadLn(a);
	ReadLn(b);
	ReadLn(c);
	ReadLn(aux_leer_logica);
	d := (lowercase(aux_leer_logica)='verdadero');
	WriteLn(a);
	WriteLn(b);
	WriteLn(c);
	WriteLn(d);
	ReadLn(a,b,c);
	ReadLn(aux_leer_logica);
	d := (lowercase(aux_leer_logica)='verdadero');
	WriteLn(a,b,c,d);
	Write('Esta linea no lleva enter al final');
End.

File: test/export/java/02-entrada-salida.java
import java.io.*;
public class sin_titulo {
	public static void main(String args[]) throws IOException {
		BufferedReader bufEntrada = new BufferedReader(new InputStreamReader(System.in));
		int a;
		double b;
		String c;
		boolean d;
		a = Integer.parseInt(bufEntrada.readLine());
		b = Double.parseDouble(bufEntrada.readLine());
		c = bufEntrada.readLine();
		d = Boolean.parseBoolean(bufEntrada.readLine());
		System.out.println(a);
		System.out.println(b);
		System.out.println(c);
		System.out.println(d);
		a = Integer.parseInt(bufEntrada.readLine());
		b = Double.parseDouble(bufEntrada.readLine());
		c = bufEntrada.readLine();
		d = Boolean.parseBoolean(bufEntrada.readLine());
		System.out.println(a+b+c+d);
		System.out.print("Esta linea no lleva enter al final");
	}
}
